includes/settings/settings-page.php

```diff
@@ -363,6 +371,21 @@ public function display_settings_page() {
 		<?php
 	}
 
+	private function render_ongoing_experiments_section() {
+		echo '<h2>' . esc_html__( 'Ongoing Experiments', 'elementor' ) . '</h2>';
+
+		echo '<table class="form-table">';
+
+		do_settings_fields( static::PAGE_ID, 'elementor-experiments-ongoing-section' );
+
+		echo '</table>';
+
+		echo '<hr>';
+
+		echo '<h2>Stable Features</h2>';
+
+	}
+
```

Comment:
Isn't it a generic page that should get its content from a dedicated file? (a file that is dedicated to the content of the experiment?).

Reply:
This is how WordPress works, I don't think we can add setting fields in a different way.
@kobizz What do you think?